app/src/main/java/openfoodfacts/github/scrachx/openfood/repositories/IProductRepository.java

```diff
@@ -48,6 +51,12 @@
 
     void saveCategories(List<Category> categories);
 
+    void deleteIngredientCascade();
+
+    void saveIngredients(List<Ingredient> ingredients);
+
+    void saveIngredient(Ingredient ingredient);
```

Comment:
Not used I guess. Could it be removed or will be used ?

Reply:
Will be used in "Diet" (and may be in other uses) because there is a lot of ingredients defined in products that are not (yet) in the taxonomy.